<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QMainWindow" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EasyList - by Willem Liu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRotate">
    <property name="title">
     <string>Rotate</string>
    </property>
    <addaction name="actionRotate"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuChecked_bottom">
    <property name="title">
     <string>Checked to bottom</string>
    </property>
    <addaction name="actionChecked_bottom"/>
   </widget>
   <widget class="QMenu" name="menuAuto_Orientation">
    <property name="title">
     <string>Auto Orientation</string>
    </property>
    <addaction name="actionAuto_Orientation"/>
   </widget>
   <widget class="QMenu" name="menuSort_A_Z">
    <property name="title">
     <string>Sort A-Z</string>
    </property>
    <addaction name="actionSort_A_Z"/>
   </widget>
   <widget class="QMenu" name="menuList">
    <property name="title">
     <string>Lists...</string>
    </property>
    <addaction name="actionLists"/>
   </widget>
   <addaction name="menuList"/>
   <addaction name="menuAuto_Orientation"/>
   <addaction name="menuSort_A_Z"/>
   <addaction name="menuChecked_bottom"/>
   <addaction name="menuRotate"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionRotate">
   <property name="text">
    <string>Rotate</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionChecked_bottom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Checked to bottom</string>
   </property>
  </action>
  <action name="actionAuto_Orientation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto Orientation</string>
   </property>
  </action>
  <action name="actionSort_A_Z">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sort A-Z</string>
   </property>
  </action>
  <action name="actionLists">
   <property name="text">
    <string>Lists...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
